### ISSUE TITLE
val_loss vs loss

### ISSUE THREAD

**Issue by muye5** [experiencor/keras-yolo3#62]
@experiencor in yolov3, the EarlyStopping、ModelCheckpoint all depend on
loss (monitor='loss')，but in yolov2，'val_loss' used.
Have you compared the difference of two settings? For I learned these two version code. (thanks for your job) and using val_loss in yolov3, the training procedure terminates faster but with lower mAP. Is this normal, thank you very much.

**Comment by muye5** [comment]
Thank you @RichardSieg  I try these two settings with VOC train / val dataset, With val_loss, after early stopping, I find val_loss will keep decreasing if I train this mode again.
